fprettify/__init__.py

```diff
@@ -1556,11 +1587,18 @@ def get_arg_parser(args):
                             help="Overrides default fortran extensions recognized by --recursive. Repeat this option to specify more than one extension.")
         parser.add_argument('--version', action='version',
                             version='%(prog)s 0.3.6')
+        parser.add_argument('--preprocessor', type=str2bool, default=False,
```

Comment:
That's a too generic argument name IMO. Let's use `--indent-fypp`, as it affects only Fypp directives (but not other type of preprocessor directives)